### DISCUSSION TITLE
MEETING: efmnhtransactions Working Group - 5 October 2022

### DISCUSSION THREAD

**Discussion by fmjjones** [fieldmuseum/EMu-Documentation#673] [MEETINGS]
# Agenda 
 [Transactions Documentation](https://github.com/fieldmuseum/EMu-Documentation/wiki/efmnhtransactions-module)

## Printed notes field
- Can we make it bigger?
- No. If more room in needed, use the Notes field on the Notes Tab with a "Kind" of Summary. Submit helpdesk request for invoice update.
- 
## Status field
 How do we indicate something was considered but ultimately rejected for a loan?
- Use Cases?

## number of Objects linked

- Upcoming development

### Flagging CITES specimens in loans (from larger CITES in Catalogue issue)

# Discussion
## Printed notes field (Mammals)
Can we make it bigger? [Should we / is there a better place to handle additional info]
Does anyone else need the field to be bigger?
Mammals LJ: JPhelps is using it for a narrative/synopsis of the whole loan and it is too small
TENTATIVE DECISION: Move synopsis to notes…+ Note Kind “Summary”
Anthro LH: Could have been helpful when transferring from the old eloans module but is not now necessary.

## How do we indicate something (an object) from a request was considered but ultimately rejected for a loan?
Would like to add a Status field added to the object table  (Anthropology)
Anthro LH - tracking loans / approval at level below “whole loan” (e.g. condition or on exhibit)
Exhibits can likewise record obj-level approval status in exhobj
SG Same for other loans? (where only part of loan-request = approved?)
Bot AB / VertPaleo AS - not frequently
Herps JM - record e.g. ‘only half requested specimens loaned’ in printed notes
JJ - track as Consultation/Info Request?
Anthro LH - setup a Transaction + immediate return for unapproved requests (since need Loan # first / for approval process)
Reasons for why not approved generally = physical documentation + Conservation record
Object table – picturing Status field on each row / open to alternatives
Caveats:
Sorting ExObj list in Events sometimes scrambles status
Alignment across Anthro + Repatriation/other depts [using Consultations]
Eg Anthro Transaction IRN 67485 - partly unapproved
TO DO/NEXT STEPS: JJ to investigate options
## Number of Objects linked
Upcoming development from Axiell
Increasing the size of tables will no longer have to be requested / be a change request.

 
### Flagging CITES specimens in loans/invoices (from larger CITES in Catalogue/taxonomy issue)
can/should come via Catalogue
Eg Invoice# Z-53222 - includes endangered taxa
TO DO [JJ to pull/prep data]
### Incoming Loans [JM + AF]
How is everyone tracking + closing incoming loans?
JM - when sending back, ‘returning loan’ can’t be linked to original incoming loan

## HOMEWORK: [all] pull together workflows for how areas document incoming loans + returning those loans
(Original working group notes may include some of that summary/info)
## HOMEWORK: JW to put in helpdesk ticket:
[report name?] - Taxon listed in wrong order
Can ‘return loan invoice’ include not-yet-returned/outstanding objects?
Example irn’s

Next regroup: ~1 month

**Comment by fmjjones** [comment]
Insects provided irn 41920 as an example of a loan return where the uncatalogued objects outgoing list does not match the incoming list and therefore obscures unreturned objects. We need a workflow to account for this in at least Insects, but maybe other collections? And are there structural changes required in Transactions Module?

**Reply by fmjjones** [reply]
The variations in the text description between Uncat Objects and Uncat Returned Objects makes it nearly impossible to reliable summarize and calculate unreturned objects (see ZenDesk ticket 57797) Can we add a Taxonomy link to the uncat objects and Returned Uncatalog objects to create a standard "parent" link?

**Comment by smochel1716** [comment]
Fishes enters incoming loans from other institutions.  Fishes doesn't make another loan for the return of that loan.
